### PR TITLE
33 create list of pools

### PR DESCRIPTION
**Pull request by ngrujicic**

### CODE REVIEW

File: src/components/organism/Table/style.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.custom-table {
+  @apply relative w-full border-separate border-spacing-y-0;
+
+  tr {
+    th {
+      @apply px-[20px] py-[24px] text-[14px] font-medium capitalize;
```
Comment (npanajotov) [inline src/components/organism/Table/style.scss]: Please use tailwind spacing instead px.

File: src/components/organism/Table/index.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      className={classNames(className, {
+        "cursor-pointer": onClick,
+      })}
+    >
+      {children}
+    </td>
+  );
+};
+
+Table.Head.displayName = "Head";
```
Comment (npanajotov) [inline src/components/organism/Table/index.tsx:90]: What is displayName means?

File: src/components/molecule/tokenAmountInput/index.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        id="token-amount"
+        allowNegative={false}
+        fixedDecimalScale
+        displayType={"input"}
+        placeholder={"0"}
+        className="w-full basis-auto bg-transparent font-inter text-[28px] font-bold text-black outline-none"
+        onFocus={() => onSetIsFocused(true)}
+        value={tokenValue}
+        onValueChange={(values) => {
+          const { floatValue } = values;
```
Comment (npanajotov) [inline src/components/molecule/tokenAmountInput/index.tsx]: You can destructuring object in function params.

File: src/components/atom/Button/style.scss [hunk 4]
```diff
@@ -4,23 +4,23 @@ button {
...
   }
 
   &.btn-primary-select {
-    @apply justify-center gap-2 rounded-full bg-pink px-3 py-1.5 font-inter text-large font-normal text-white hover:bg-black;
+    @apply justify-center gap-2 rounded-full bg-pink px-[12px] py-[3.5px] font-inter text-large font-normal text-white hover:bg-black;
   }
 
   &.btn-secondary-select {
-    @apply justify-center gap-2 rounded-full bg-purple-200 px-6 py-4 font-inter font-normal text-black hover:bg-black hover:text-white;
+    @apply justify-center gap-2 rounded-full bg-purple-200 px-[12px] py-[6px] font-inter font-normal text-black hover:bg-black hover:text-white;
```
Comment (npanajotov) [inline src/components/atom/Button/style.scss]: Please use tailwind spacing instead px.

File: src/components/molecule/tokenAmountInput/index.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+      <label htmlFor="token-amount" className="absolute top-4 text-[11px] font-normal text-[#797a7e]">
+        You pay
+      </label>
+      <NumericFormat
+        id="token-amount"
+        allowNegative={false}
+        fixedDecimalScale
+        displayType={"input"}
+        placeholder={"0"}
+        className="w-full basis-auto bg-transparent font-inter text-[28px] font-bold text-black outline-none"
```
Comment (ra79sta) [inline src/components/molecule/tokenAmountInput/index.tsx]: Font is inter by default, font size is  in tailwind config

File: src/components/molecule/tokenAmountInput/index.tsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  ref: React.ForwardedRef<HTMLDivElement>
+) {
+  return (
+    <div
+      ref={ref}
+      className={`relative flex h-[103px] w-[424px] items-center justify-start gap-2 rounded-[8px] bg-[#F3F5FB] px-4 py-6 ${
+        isFocused ? "border border-solid border-[#E6007A]" : null
+      }`}
+    >
+      <label htmlFor="token-amount" className="absolute top-4 text-[11px] font-normal text-[#797a7e]">
```
Comment (ra79sta) [inline src/components/molecule/tokenAmountInput/index.tsx]: Font size is in tailwind config

File: src/components/organism/Table/index.tsx (hunk 2 above)
Comment (ngrujicic) [reply src/components/organism/Table/index.tsx:90]: i had an error in the component and i had to fix it with this - here is the error and explenation - https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/display-name.md

File: src/components/organism/Table/style.scss (hunk 1 above)
Comment (ngrujicic) [reply src/components/organism/Table/style.scss]: fixed

File: src/components/atom/Button/style.scss (hunk 4 above)
Comment (ngrujicic) [reply src/components/atom/Button/style.scss]: fixed

File: src/components/molecule/tokenAmountInput/index.tsx (hunk 5 above)
Comment (ngrujicic) [reply src/components/molecule/tokenAmountInput/index.tsx]: fixe, merged from develop

File: src/components/molecule/tokenAmountInput/index.tsx (hunk 6 above)
Comment (ngrujicic) [reply src/components/molecule/tokenAmountInput/index.tsx]: fixed merged from develop

File: src/components/molecule/tokenAmountInput/index.tsx (hunk 3 above)
Comment (ngrujicic) [reply src/components/molecule/tokenAmountInput/index.tsx]: fixed

File: src/pages/PoolsPage/PoolTable.tsx [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+    tokenAIcon: <DotToken width={36} height={36} />,
+    tokenBIcon: <TestToken width={36} height={36} />,
+    totalTokensLocked: {
+      tokenA: "234.42",
+      tokenB: "436.42",
+      tokenAIcon: <DotToken width={16} height={16} />,
+      tokenBIcon: <TestToken width={16} height={16} />,
+    },
+  },
+];
```
Comment (markoMVP85) [inline src/pages/PoolsPage/PoolTable.tsx:107]: Could u wrap this with Promise? We need that cuz of the load state.
smth like this
const fetchData = () => {
  return new Promise((resolve) => {
    setTimeout(() => {
      const mockData = [
        { id: 1, title: 'Mock Item 1' },
        { id: 2, title: 'Mock Item 2' },
        { id: 3, title: 'Mock Item 3' },
      ];
      resolve(mockData);
    }, 1000); // Simulating a delay of 1 second
  });
};

File: src/pages/PoolsPage/PoolTable.tsx [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+
+const PoolTable = () => {
+  return (
+    <Table className="w-full max-w-[1280px] font-inter">
```
Comment (markoMVP85) [inline src/pages/PoolsPage/PoolTable.tsx]: Maybe we should remove the width from here and place it in the parent component. You can make a "container" in the parent component and just call the "PoolTable".

File: src/pages/PoolsPage/index.tsx [hunk 9]
```diff
@@ -1,7 +1,9 @@
+import PoolTable from "./PoolTable";
+
 const PoolsPage = () => {
   return (
-    <div className="flex justify-center">
-      <h1>Pool list</h1>
+    <div className="w-full px-28">
```
Comment (markoMVP85) [inline src/pages/PoolsPage/index.tsx:5]: We should use here width logic rather than use it in the table component.

Comment (ngrujicic) [reply src/pages/PoolsPage/index.tsx:5]: done

File: src/pages/PoolsPage/PoolTable.tsx (hunk 8 above)
Comment (ngrujicic) [reply src/pages/PoolsPage/PoolTable.tsx]: done

File: src/pages/PoolsPage/PoolTable.tsx (hunk 7 above)
Comment (ngrujicic) [reply src/pages/PoolsPage/PoolTable.tsx:107]: done

**Review by markoMVP85 — APPROVED** [review]

**Pull request merged**